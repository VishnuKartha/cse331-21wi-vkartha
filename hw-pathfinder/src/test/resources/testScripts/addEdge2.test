# Creates a single graph, adds two nodes, and adds two edges between the two nodes.

# Create a Graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add two edges between the two nodes
AddEdge graph1 n1 n2 1010.00
AddEdge graph1 n2 n1 2020.00