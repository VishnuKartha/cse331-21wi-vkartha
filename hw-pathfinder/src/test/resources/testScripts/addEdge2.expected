# Creates a single graph, adds two nodes, and adds two edges between the two nodes.

# Create a Graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add two edges between the two nodes
added edge 1010.000 from n1 to n2 in graph1
added edge 2020.000 from n2 to n1 in graph1